### PR TITLE
Modified sensor_fetch file to save from DB

### PR DESCRIPTION
**Pull request by WarClans612**

### CODE REVIEW

File: sensor_fetch/util.py [hunk 1]
```diff
@@ -26,6 +26,8 @@ def save_data_into_db(data, collection_name):
     db = connect_to_database()
     collect = db[collection_name]
     collect.insert(data)
+    #temporary return
```
[the remark this answers is not in the document]
Comment (WarClans612) [reply sensor_fetch/util.py]: The pymongo provide way to count the number of data that is successfully inserted. I am thinking of implementing it on later date

[the remark this answers is not in the document]
Comment (Billy4195) [reply sensor_fetch/util.py]: I thinks this should be done in this commit, don't leave some bugs for later to fix

[the remark this answers is not in the document]
Comment (WarClans612) [reply sensor_fetch/util.py]: Many implementation of determining the trueness for the insert is done as the one in this commit. The counting is just an additional layer of checking to be implemented if there are sufficient time to do it

[the remark this answers is not in the document]
Comment (Billy4195) [reply sensor_fetch/util.py]: But I can't see any check of the insert operation is done without problem, or I just miss it.
Could you point me out where is the check?

[the remark this answers is not in the document]
Comment (WarClans612) [reply sensor_fetch/util.py]: I see your point now. I will check it soon

**Comment by WarClans612** [comment]
Updated

File: sensor_fetch/util.py [hunk 2]
```diff
@@ -36,12 +40,20 @@ def save_station_to_db(data, station_name):
     db = connect_to_database()
     db.drop_collection(station_name)
     collect = db[station_name]
-    collect.insert_many(data)
+    insert_status = collect.insert_many(data)
+    if len(insert_status.inserted_ids) == len(data):
+        return True
+    else:
+        return False
```
Comment (Billy4195) [inline sensor_fetch/util.py]: The `save_data_into_db()` and `save_station_to_db()` is almost the same function, please combine it into one function.

**Comment by WarClans612** [comment]
Updated

**Pull request merged**